src/controllers/commands/builtin/twitch/Announce.cpp

```diff
@@ -25,25 +26,38 @@ QString sendAnnouncement(const CommandContext &ctx)
         return "";
     }
 
+    QString colorStr = "";
+    if (color != HelixAnnouncementColor::Primary)
+    {
+        colorStr =
+            QString::fromStdString(
+                std::string{
+                    magic_enum::enum_name<HelixAnnouncementColor>(color)})
+                .toLower();
+    }
+
     if (ctx.words.size() < 2)
     {
         ctx.channel->addMessage(makeSystemMessage(
-            "Usage: /announce <message> - Call attention to your "
-            "message with a highlight."));
+            QString("Usage: /announce%1 <message> - Call attention to your "
+                    "message with a %1%2highlight.")
+                .arg(colorStr)
+                .arg(color == HelixAnnouncementColor::Primary ? "" : " ")));
```

Comment:
nit: do an if on the color. this makes the message formatting cleaner & moves the colorStr logic into the same color check, something along the lines of
```
if color == Primary:
  msg = "Usage: /announce <message> - Call attention to your message with a highlight"
else:
  colorstr = magic_enum(...)
  msg = QString("Usage: /announce%1 <message> - Call attention to your message with a %1 highlight", colorstr)

ctx.channel->addMessage(makeSystemMessage(msg));
```